components/appbar/position.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Position
+page_title: AppBar Position
+description: Position settings of the AppBar for Blazor.
+slug: appbar-position
+tags: telerik,blazor,appbar,navbar,position
+published: True
+position: 35
+---
+
+# Position Settings
+
+This article outlines the available AppBar parameters, which control its position.
+
+>note Read the [CSS positioning MDN documentation article](https://developer.mozilla.org/en-US/docs/Web/CSS/position) to get a better understanding of how the AppBar component positioning work.
+
+## Position
+
+The `Position` parameter accepts a member of the `AppBarPosition` enum and sets the `top` and `bottom` CSS properties:
+
+| Enum members | Description |
```

Comment:
Either use singular or plural in both columns. I believe singular is the more sensible approach.